engine/apps/api/tests/test_schedules.py

```diff
@@ -219,7 +219,7 @@ def test_get_list_schedules_pagination(
     client = APIClient()
 
     schedule_list_url = reverse("api-internal:schedule-list")
-    absolute_url = create_engine_url(schedule_list_url, override_base="http://testserver")
```

Comment:
this should be fine. `override_base` has no effect here because the _actual_ absolute URL that is being generated is overriden here on the actual `request` object itself in `PathPrefixedCursorPagination.paginate_queryset`:
```py3
class PathPrefixedPagination(PageNumberPagination):
    def paginate_queryset(self, queryset, request, view=None):
        request.build_absolute_uri = lambda: create_engine_url(request.get_full_path())
        return super().paginate_queryset(queryset, request, view)
```